gulp-core-build/src/tasks/CleanTask.ts

```diff
@@ -59,8 +58,17 @@ export class CleanTask extends GulpTask<void> {
       }
     }
 
-    del(cleanPaths)
-      .then(() => completeCallback())
-      .catch((error) => completeCallback(error));
+    return Promise.all(
```

Comment:
It would be better to run these promises serially rather than in parallel.  If you start 20 different "rm -Rf *" processes, it's going to thrash the disk.